services/rollout-service/pkg/versions/versions_test.go

```diff
@@ -45,14 +45,15 @@ type step struct {
 }
 
 type expectedVersion struct {
-	Revision        string
-	Environment     string
-	Application     string
-	DeployedVersion uint64
-	DeployTime      time.Time
-	SourceCommitId  string
-	Metadata        metadata.MD
-	IsProduction    bool
+	Revision         string
+	Environment      string
+	Application      string
+	DeployedVersion  uint64
+	DeployTime       time.Time
+	SourceCommitId   string
+	OverviewMetadata metadata.MD
```

Comment:
it looks like you're only using `OverviewMetadata` and `VersionMetadata` in the tests. Sounds a bit like a changed file was not pushed?

Reply:
No, it is actually checked in the test. However, my change made it always empty